data-collections-exercises.md

```diff
@@ -15,83 +15,91 @@ Access the second value in `planeteers`.
 Add "Heart" to the end of `planeteers`.
 
 ```rb
-# Your answer here
+planeteers.push("Heart")
 ```
 
 Remove "Captain Planet" from the array (without using a method).
 
 ```rb
-# Your answer here
+planeteers.delete("Captain Planet")
```

Comment:
To remove it without using a method we can subtract the value like this

planeteers - ["Captain Planet"]